conda/reporters.py

```diff
@@ -25,7 +25,7 @@
 logger = logging.getLogger(__name__)
 
 
-@lru_cache(maxsize=None)
+@cache
```

Comment:
This returns the same as `@lru_cache(maxsize=None)` ([reference](https://docs.python.org/3/library/functools.html#functools.cache))